### PR TITLE
feat: Migrate API to use python micro-generator

### PR DESCRIPTION
**Pull request by hkdevandla**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-language/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by hkdevandla** [comment]
Please hold on with the review. Fixing errors. Thanks!

File: synth.py [hunk 1]
```diff
@@ -38,23 +38,10 @@
...
-'''__doc__="""Represents the input to API methods.'''
-)
-s.replace(
-    f"google/cloud/**/gapic/language_service_client.py",
-    r"types\.EncodingType",
-    "enums.EncodingType",
-)
-
 # TODO(busunkim): Use latest sphinx after microgenerator transition
 s.replace("noxfile.py", """['"]sphinx['"]""", '"sphinx<3.0.0"')
```
Comment (software-dov) [inline synth.py:46]: I think these lines can be removed.

File: samples/snippets/api/analyze_test.py [hunk 2]
```diff
@@ -18,8 +18,7 @@
 
 def test_analyze_entities():
     result = analyze.analyze_entities(
-        "Tom Sawyer is a book written by a guy known as Mark Twain."
-    )
+        request = {'document': "Tom Sawyer is a book written by a guy known as Mark Twain."})
```
Comment (software-dov) [inline samples/snippets/api/analyze_test.py:21]: Though his mind is not for rent, don't put him down as arrogant!

File: samples/v1/language_sentiment_gcs.py [hunk 3]
```diff
@@ -43,7 +41,7 @@ def sample_analyze_sentiment(gcs_content_uri):
     # gcs_content_uri = 'gs://cloud-samples-data/language/sentiment-positive.txt'
 
     # Available types: PLAIN_TEXT, HTML
-    type_ = enums.Document.Type.PLAIN_TEXT
+    type_ = language_v1.enums.Document.Type.PLAIN_TEXT
```
Comment (software-dov) [inline samples/v1/language_sentiment_gcs.py:44]: There isn't a generated `enums` submodule any more. Does the sample run?

**Review by software-dov — COMMENTED** [review]
Looks good, just one minor nit and a concern about samples.

**Comment by software-dov** [comment]
I think the reason the other samples tasks are failing is because of the `enums` submodule; this was removed in the microgenerator surface.

File: samples/v1/language_sentiment_gcs.py (hunk 3 above)
Comment (hkdevandla) [reply samples/v1/language_sentiment_gcs.py]: My bad. Fixed now. Moreover these samples (in samples/v1/) are pre-existing ones and not from python-doc-samples repo.

File: synth.py (hunk 1 above)
Comment (hkdevandla) [reply synth.py]: Done.

**Review by software-dov — APPROVED** [review]

**Review by busunkim96 — APPROVED** [review]

**Pull request merged**